src/js/components/molecules/cards/card-article/card-article.scss

```diff
@@ -29,9 +36,23 @@
     margin-bottom: $buildingUnit * 2;
 }
 
-.card-article__description {
-    font-size: 14px;
-    margin-bottom: $buildingUnit;
-    padding-left: $buildingUnit;
-    padding-right: $buildingUnit;
+.card-article__list {
```

Comment:
`list` is too generic I think… `labels-list` maybe? Bit longer but more descriptive.

Reply:
⚡️